A3A/addons/core/Params.hpp

```diff
@@ -512,6 +512,14 @@ class Params
         texts[] = {$STR_antistasi_dialogs_generic_button_no_text, $STR_antistasi_dialogs_generic_button_yes_text};
         default = 0;
     };
+    class enableSpectrumDevice
+    {
+        attr[] = {"server"};
```

Comment:
This has `attr[] = {"server"}`, but you're also checking it in `initClient`. Does it still work?

Reply:
Does work, though haven't tested it on dedicated server

Reply:
Would probably be better to just remove this if the client is checking it too
```suggestion
    class enableSpectrumDevice
    {
```